crates/uv/src/commands/project/lock.rs

```diff
@@ -528,7 +528,11 @@ async fn do_lock(
                     .collect(),
                 dev,
                 source_trees,
-                None,
+                workspace
+                    .pyproject_toml()
+                    .project
+                    .as_ref()
+                    .map(|proj| proj.name.clone()),
```

Comment:
Note: drive-by fix, I noticed this detail was missing when looking into a derivation tree result from PubGrub. I *think* in practice this should be a no-op change for any observable behavior, but I'll CI check that.